lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -957,7 +960,7 @@ bool TypeLoweringVisitor::visitDecl(FExtModuleOp extModule) {
     newArgDirections.push_back(port.direction);
     newArgNames.push_back(port.name);
     newPortTypes.push_back(TypeAttr::get(port.type));
-    newArgSyms.push_back(port.sym ? port.sym : cache.sEmpty);
+    newArgSyms.push_back(port.sym ? port.sym : cache.symEmpty);
```

Comment:
Is InnerSymAttr("") different than InnerSymAttr()?  And can this just be `port.sym`?

Reply:
You are absolutely right, just using `port.sym`